Add Drain Cleaner installation files to the operators repo (#5560)
strimzi/strimzi-kafka-operator: packaging/install/drain-cleaner/kubernetes/README.md

# Installing Strimzi Drain Cleaner on Kubernetes without CertManager

> Strimzi Drain Cleaner is using `ValidationWebhook` to receive information about Strimzi pods being evicted.

Reviewer: ```suggestion
Strimzi Drain Cleaner uses a `ValidationWebhook` to receive information about Strimzi pods being evicted.
```

Kubernetes requires that `ValidationWebhooks` are secured using TLS.

> So the web-hook service need to have HTTPS support.

Reviewer: ```suggestion
So the web-hook service needs to have HTTPS support.
```
